api/oas.yaml

```diff
@@ -0,0 +1,897 @@
+openapi: 3.0.1
+info:
+  title: Stargate REST API reference
+  description: "The Stargate REST API provides CRUD operations on table data managed by Stargate."
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 2.0.0
+externalDocs:
+  description: Reference Document
+  url: http://shorturl.at/hnqIQ
+servers:
+  - url: https://api.stargate.io/rest/v2
+tags:
+  - name: health
+    description: "Health checks"
+  - name: schemas
+    description: "Interact with keyspaces and tables in your database"
+  - name: data
+    description: "Add, update, and delete rows in your database"
+paths:
+  /keyspaces/{keyspace-id}/{table-id}:
+    get:
+      tags:
+        - data
+      summary: search a table
+      parameters:
+        - $ref: "#/components/parameters/pretty"
+        - $ref: "#/components/parameters/keyspace-id"
+        - $ref: "#/components/parameters/table-id"
+        - $ref: "#/components/parameters/where"
+        - $ref: "#/components/parameters/fields"
+        - $ref: "#/components/parameters/page-size"
+        - $ref: "#/components/parameters/page-state"
+        - $ref: "#/components/parameters/sort"
+        - $ref: "#/components/parameters/raw"
+      responses:
+        200:
+          description: ""
+          content:
+            application/json:
+              examples:
+                default:
+                  value:
+                    {
+                      "count": 1,
+                      "pageState": "2f4acd34",
+                      "data":
+                        [{ "title": "Hello World", "author": "Cliff Wicklow" }],
+                    }
```

Comment:
These were all supposed to have proper OAS objects rather than raw json. Otherwise this breaks client generators.